apps/web/pages/settings/my-account/calendars.tsx

```diff
@@ -51,7 +62,11 @@ const CalendarsView = () => {
 
   return (
     <>
-      <Meta title="Calendars" description="Configure how your event types interact with your calendars" />
+      <Meta
+        title="Calendars"
+        description="Configure how your event types interact with your calendars"
+        CTA={<AddCalendarButton />}
+      />
```

Comment:
This `<Meta />` component is present on every settings page. This is used to programmatically set the title and description of the header sections of each page.

Since we want to add the CTA to this section, it makes sense to pass it along with the title and description.

Ex.
![image](https://user-images.githubusercontent.com/65426560/196808552-4dda8a35-961a-488f-8ec3-7aff3720e906.png)